src/__snapshots__/class.test.js.snap

```diff
@@ -105,3 +105,47 @@ Array [
   ],
 ]
 `;
+
+exports[`ctx.iopipe.metric adds metrics to the custom_metrics array 1`] = `
+Array [
+  Object {
+    "n": undefined,
+    "name": "metric-1",
+    "s": "foo",
+  },
+  Object {
+    "n": undefined,
+    "name": "metric-2",
+    "s": "true",
+  },
+  Object {
+    "n": undefined,
+    "name": "metric-3",
+    "s": "{\\"ding\\":\\"dong\\"}",
+  },
+  Object {
+    "n": undefined,
+    "name": "metric-4",
+    "s": "[\\"whoa\\"]",
+  },
+  Object {
+    "n": 100,
+    "name": "metric-5",
+    "s": undefined,
+  },
+]
+`;
+
+exports[`ctx.iopipe.tag adds tags to the custom_metrics array 1`] = `
+Array [
+  Object {
+    "name": "tag-1",
+  },
+  Object {
+    "name": "2",
+  },
+  Object {
+    "name": "[object Object]",
```

Comment:
Is this expected? If you use `JSON.stringify` we could support objects, `undefined` will return `undefined` though so we could do an additional fallback to `String(val)` perhaps?

Reply:
It is expected, it's presuming you pass a string to `.tag`, but I expected a discussion here about what we prefer.